model/Core/Classes/ElementCollection.md

```diff
@@ -13,6 +13,9 @@ An ElementCollection is a collection of Elements, not necessarily with unifying
 Note that all ElementCollections must conform to the core profile even if the core profile is no specified in the profileConformance property.
 If the profileConformance property is not provided, core is to be assumed as the default.
 
+**Constraints**
+If the ElementCollection has at least 1 element, it must also have at least 1 rootElement
```

Comment:
Is that the way how we encode constraints?

This feels like a step in the direction of `filesAnalyzed` and it introduces more complexity. Not sure if I like that.

Reply:
I thought there was a better way to encode the constraints, but I was unable to find an example, so I put this in as a placeholder for now